src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -37,15 +38,15 @@ public function __construct($name = 'debug:twig')
     /**
      * Sets the twig environment.
      *
-     * @param \Twig_Environment $twig
+     * @param Environment $twig
      */
-    public function setTwigEnvironment(\Twig_Environment $twig)
+    public function setTwigEnvironment(Environment $twig)
```

Comment:
@javiereguiluz This would not be PSR-4 compliant. In §2.2.1, "The fully qualified class name MUST have a top-level namespace name, also known as a “vendor namespace”.
http://www.php-fig.org/psr/psr-4/#specification